frontend/web/components/InfoMessage.js

```diff
@@ -8,7 +8,11 @@ export default class InfoMessage extends PureComponent {
   static displayName = 'InfoMessage'
 
   handleOpenNewWindow = () => {
-    window.open(this.props.url, '_blank')
+    if (this.props.goToIntegrations) {
```

Comment:
Let's come up with a better solution than adding goToIntegrations as a prop to a generic component like InfoMessage, it doesn't make sense.